changelog/22810.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+ui: Fixes old pki's filter and search roles page bug 
```

Comment:
Maybe let's update the comment to something more generic? `Fix filter roles when API does not return ID` ?